universe/Fleet.cpp

```diff
@@ -990,6 +990,11 @@ void Fleet::MovementPhase() {
                 if (supply_unobstructed_systems.find(SystemID()) != supply_unobstructed_systems.end()) {
                     m_arrival_starlane = SystemID();//allows departure via any starlane
                 }
+
+                // Add current system to the start of any existing route for next turn
```

Comment:
the description refers to blockades, but this seems potentially more generally. is this sure to not cause a system to be unnecessarily added to the route in the general case?